dashboard/baserpc.py

```diff
@@ -351,15 +351,6 @@ async def get_server(self, userid: int, serverid: int):
         else:
             vl = "Unknown"
 
-        region = getattr(guild.region, "name", guild.region)
-        parts = region.split("_")
-        for i, p in enumerate(parts):
-            if p in ["eu", "us", "vip"]:
-                parts[i] = p.upper()
-            else:
-                parts[i] = p.title()
-        region = " ".join(parts)
-

```

Comment:
Region is no longer part of the guild object. Maybe replace region with the servers locale?